samples/vsphere/sso/external_psc_sso_workflow.py

```diff
@@ -82,7 +82,8 @@ def options(self):
         self.args = self.argparser.parse_args()
 
     def setup(self):
-        self.lswsdl = self.args.lswsdl
```

Comment:
Thanks Tianhao. Done the changes, squashed all commits and rebased.

Please let me know if more changes required